README.md

```diff
@@ -7,15 +7,15 @@ javascript objects in both client and server-side environments.
 [![build status](https://secure.travis-ci.org/kofrasa/mingo.png)](http://travis-ci.org/kofrasa/mingo)
 
 ## Dependencies
-[underscore](https://github.com/jashkenas/underscore)
+[lodash](https://github.com/lodash/lodash/)
 
 ## Installing
 ```$ npm install mingo```
 
 In browser
 ```html
-<!-- DO NOT FORGET Underscore -->
-<script type="text/javascript" src="./underscore-min.js"></script>
+<!-- DO NOT FORGET LODASH -->
+<script type="text/javascript" src="./lodash.min.js"></script>
 <script type="text/javascript" src="./mingo.min.js"></script>
```

Comment:
The min+gzip size should be smaller with Lodash (at least my test bundle was) by 0.5 kb. It not it will be by next version bump (depends if my test bundle was based on work-in-progress bits).